crates/apollo-compiler/tests/validation/ignore_builtin_redefinition.rs

```diff
@@ -0,0 +1,256 @@
+use apollo_compiler::schema::SchemaBuilder;
+use apollo_compiler::Schema;
+use expect_test::expect;
+
+#[test]
+fn handles_built_in_scalar_redefinition() {
+    let schema = r#"
+  scalar String
```

Comment:
So we are extra sure, I added all the scalar redefinitions and all the type redefinitions (below).